common/src/test/java/org/apache/sedona/common/raster/RasterOutputTest.java

```diff
@@ -68,4 +70,37 @@ public void testAsGeoTiffWithCompressionTypes() throws IOException {
         assertEquals(rasterOg.getEnvelope().toString(), rasterNew.getEnvelope().toString());
         assert(rasterBytes1.length > rasterBytes2.length);
     }
+
+    @Test
+    public void testAsMatrixBand() {
```

Comment:
Please also add tests with a real raster that has multiple bands.

Reply:
I've added a test testing other band for a test raster. The real test rasters I have are too huge and I cannot hardcode an expectedValue into the test.